src/server/handlers/metrics.ts

```diff
@@ -26,7 +26,11 @@
 import Metrics from '@mojaloop/central-services-metrics'
 import { Request, ResponseToolkit, ResponseObject } from '@hapi/hapi'
 
-export function get (_: Request, h: ResponseToolkit): ResponseObject {
+function get (_: Request, h: ResponseToolkit): ResponseObject {
```

Comment:
I would keep export here to have the same consistency wich we keep in other handlers

Reply:
yup agreed